terraform/environments/hmpps-oem/locals.tf

```diff
@@ -95,6 +146,8 @@ locals {
     }
   }
   baseline_iam_roles = {
+    # allow EC2 instance profiles ability to assume this role
+    EC2OracleEnterpriseManagementSecretsRole = module.baseline_presets.iam_roles_all.EC2OracleEnterpriseManagementSecretsRole
```

Comment:
If we set enable_ec2_oracle_enterprise_managed_server = true  then we don't need this.